### DISCUSSION TITLE
Bug: reusing component causes `redeclaration of import` error

### DISCUSSION THREAD

**Discussion by PindaPixel** [posva/unplugin-vue-router#262] [Questions and Help]
I'm using `extendRoute` to add catch all route that renders a component that throws an error, so a parent component can handle routing related errors with `onErrorCaptured`.

Each time it's used, it redeclares it with `default as _page_<number>`, but the number incrementing seems to be acting out.

```ts
import _page_0 from "/src/Modules/Base/BaseLayout.vue"
import { default as _page_1, default as _page_3, default as _page_4, default as _page_5 } from "/src/Router/Components/NotFound.vue"
import _page_2 from "/src/Modules/Account/Pages/AccountLayout.vue"
import _page_3 from "/src/Modules/Reports/Pages/ReportsLayout.vue"
import _page_4 from "/src/Modules/Account/Pages/Login@/LoginLayout.vue"
```

I don't have time to create a reproduction right now, if needed I can create on at a later time.

**Comment by posva** [comment]
I don't understand what you mean. A repro would help

**Reply by PindaPixel** [reply]
`_page_3` and `_page_4` are defined twice because of the re-usage of the `NotFound` component. I'll make a repro.

**Reply by PindaPixel** [reply]
Done: https://github.com/PindaPixel/vue-dir-routing
Started happening as soon as I added [this line](https://github.com/PindaPixel/vue-dir-routing/blob/main/config/routing.ts#L12). I imagine the more folders I add the more duplicated imports I'll get.

**Comment by posva** [comment]
Thanks again!
